src/SmartComponents/Actions/ListActions.js

```diff
@@ -47,21 +42,21 @@ class ListActions extends Component {
     }
 
     componentDidMount() {
-        const response = JSON.parse(JSON.stringify(mockData));
+        const response = this.props.AdvisorStore.mediumRiskRules;
 
         let rule = response.rules.find(obj => {
             return obj.rule_id === this.props.match.params.id;
         });
         this.setState({ rule });
 
-        const response2 = JSON.parse(JSON.stringify(impactedSystemsData));
+        const response2 = this.props.AdvisorStore.impactedSystems;
```

Comment:
I think this exceeds the scope of this  pr, trying to keep things very specific here.

Reply:
Why not use `mediumRiskRules` and `impactedSystems` for now instead of `response` and `response2`? I think it makes it incredible easier to understand what is going on.

Reply:
I agree very much so, but again, scope of the pr, this work isn't focused on overhauling the components, its adding the redux store, ensuring components consume it